src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DebuggerDisplayAttributeTests.cs

```diff
@@ -275,6 +275,52 @@ .maxstack  2
             });
         }
 
+        [Fact]
+        public void Usings2()
+        {
+            var source =
+@"using System.Diagnostics;
+using A = int;
+using B = (int, int);
+using C = int[];
+namespace N
+{
+    [DebuggerDisplay(""{typeof(A) ?? typeof(B) ?? typeof(C)}"")]
+    class D
+    {
+        void M()
+        {
+        }
+    }
+}";
+            var compilation0 = CreateCompilation(source, options: TestOptions.DebugDll, assemblyName: ExpressionCompilerUtilities.GenerateUniqueName());
+            compilation0.VerifyDiagnostics(
+                // (2,1): hidden CS8019: Unnecessary using directive.
```

Comment:
Why does this report unnecessary using?

Reply:
because they're not used.  This is a debugger-display-string.  So it's just a string.  The compiler has no concept that the string itself will end up using the aliases (That issue exists today already for normal aliases).  Not sure if we want to change anything here....

Reply:
Oh, I see. Makes sense